t04_refactored.py

```diff
@@ -146,9 +146,42 @@ def team_5_adv():
 
 ###################################################################################
 
+
 def team_6_adv():
-    pass
-    # TODO Add your code here
+    '''
+    Function that continues on which path to take in the cave!
+    '''
+
+    path = (input
+            ("Which path will you take? [Left/Middle/Right]"))
+
+    second_chance = (input
+                     ("Are you sure? [Yes/No]"))
+    if second_chance == "Yes":
+        path = input(
+            'Which path will you take? [Left/Middle/Right]')
+
+    if path == "Left":
+        # Wrong Path
+        print("Sss....sss...sss...")
+        sleep(delay)
+        print("You have encountered a python.")
+        print("Oh no! The python has bitten you.")
+        print("You are now slowly dying from the poison.")
+        dead = True
+        kill_if_dead(dead)
```

Comment:
The original prompt asked you to give the second chance **when the user is about to die** (not before). I would move your code for second chance into the left path, and only `kill_if_dead()` there. Also, the second chance was supposed to use a number (e.g., Pick a number between 1 and 10).